test/run_tests.h

```diff
@@ -19,3 +19,4 @@ void test_mainloop(void);
 void test_block_layout(void);
 void test_flex_layout(void);
 void test_widget_rect(void);
+void test_clipboard(void);
```

Comment:
A newline is required at the end of file.

![image](https://user-images.githubusercontent.com/1730073/141685033-8ac7a128-c542-4298-8fad-c6a85af397c4.png)